ansys/dpf/core/server_types.py

```diff
@@ -774,8 +778,14 @@ def __init__(self,
                     f"DPF directory not found at {os.path.dirname(path)}"
                     f"Unable to locate the following file: {path}")
             raise e
-        data_processing_capi.DataProcessingCAPI.data_processing_initialize_with_context(1, None)
         self.set_as_global(as_global=as_global)
+        try:
+            self._base_service.apply_context(server_context.SERVER_CONTEXT)
+        except errors.DpfVersionNotSupported:
+            self._base_service.initialize_with_context(
+                server_context.AvailableServerContexts.premium
```

Comment:
@cbellot000 Why premium in this case?

Reply:
@PProfizi premium was the old default (equivalent to standalone)